src/ocm/components/clusterConfiguration/ArmCheckbox.tsx

```diff
@@ -20,6 +20,8 @@ const getLabel = (openshiftVersion: string) => {
     <>
       Use arm64 CPU architecture{' '}
       <PopoverIcon
+        component="a"
```

Comment:
Why not using `variant="plain"` instead?

Also looking at patternfly examples, I am seeing that they're not using actual `Button` component but rather define a custom `<button>` for form label help popovers. I'd like to avoid this if possible but a solution could be to create a dedicated component for form label help popovers.
Also note that our `PopoverIcon` component does not set `HelpIcon`'s `noVerticalAlign` to true whereas the Patternfly example does that.

Reply:
I can't find the examples using the `button` element, could you please share the link?
I'm seeing here they are using `Button` component https://www.patternfly.org/v4/components/popover